test/e2e/apimachinery/resource_quota.go

```diff
@@ -915,6 +918,58 @@ var _ = SIGDescribe("ResourceQuota", func() {
 			framework.Failf("Expected `not found` error, got: %v", err)
 		}
 	})
+
+	ginkgo.It("should manage the lifecycle of a ResourceQuota", func() {
+		client := f.ClientSet
+		ns := f.Namespace.Name
+
+		rqName := "e2e-quota-" + utilrand.String(5)
+		label := map[string]string{"e2e-rq-label": rqName}
+		labelSelector := labels.SelectorFromSet(label).String()
+
+		ginkgo.By("Creating a ResourceQuota")
+		resourceQuota := &v1.ResourceQuota{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   rqName,
+				Labels: label,
+			},
+			Spec: v1.ResourceQuotaSpec{
+				Hard: v1.ResourceList{},
+			},
+		}
+		resourceQuota.Spec.Hard[v1.ResourceCPU] = resource.MustParse("1")
+		resourceQuota.Spec.Hard[v1.ResourceMemory] = resource.MustParse("500Mi")
+		_, err := createResourceQuota(client, ns, resourceQuota)
+		framework.ExpectNoError(err)
+
+		ginkgo.By("Getting a ResourceQuota")
```

Comment:
Is lifecycle tests for endpoints mainly mean, getting, creating, deleting ? Is this in addition to functionality testing which is already covered ?

Reply:
Dont follow what value do these add in terms of e2e coverage

Reply:
This e2e test was created with the aim of promoting it to conformance once it proved to be flakefree for 2 weeks. 
The Comformance test would cover the outstanding endpoints listCoreV1ResourceQuotaForAllNamespaces, patchCoreV1NamespacedResourceQuota and
deleteCoreV1CollectionNamespacedResourceQuota.
We are over 90% though dealing with all old conformance tech debt. 
Once that is completed it would make sense to do a complete clean-up of all tests per resource, exisising all the endpoints options in a single test.